lib/matplotlib/backends/backend_svg.py

```diff
@@ -26,196 +26,120 @@
 from matplotlib.transforms import Affine2D, Affine2DBase
 from matplotlib import _png
 
-from xml.sax.saxutils import escape as escape_xml_text
+from xml.etree import ElementTree as etree
 
 backend_version = __version__
 
-# ----------------------------------------------------------------------
-# SimpleXMLWriter class
```

Comment:
The reason we use this rather than the Python library's built-in TreeBuilder is due to streaming.  The changes here build an XML tree in memory, and then write it out.  That, at least when I originally wrote this, was much slower and less memory efficient, particularly for large paths.  There's a very good reason this XMLWriter class was used, and I think "because Python has a tree builder in the standard library" is not a good enough reason to completely change the approach here.